### DISCUSSION TITLE
Heatmap of Top Genes

### DISCUSSION THREAD

**Discussion by helpmeplsxoxo** [GreenleafLab/ArchR#1175] [Questions / Documentation]
I have been using the plotMarkerHeatmap function to plot the heatmap for my ATAC data, but it is very hard to interpret with so many genes. How can I only make a heatmap with the top 10 genes per cluster? 

Thanks!

**Comment by rcorces** [comment]
I'm not sure this is what you are looking for but the `nPrint` parameter sounds close:

<html>
<body>


nPrint | If provided seMarker is from "GeneScoreMatrix" print the top n genes for each group based on how uniquely up-regulated the gene is.
-- | --



</body>
</html>

If thats not what you mean, you will likely need to plot the heatmap manually.

**Reply by rcorces** [reply]
or trim your seMarker to only have the genes you want to plot

**Reply by helpmeplsxoxo** [reply]
Yes, I think I need to trim my seMarker. Is there a specific argument in the GetMarkerFeatures function to use to trim it?

**Reply by rcorces** [reply]
No you would trim it manually using standard `SummarizedExperiment` manipulations.
